### PR TITLE
Feature/tomte 0.2.2

### PR DESCRIPTION
**Pull request by solarw**
## Fixes

https://github.com/valory-xyz/open-autonomy/issues/1801
#1368

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [x] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/valory/skills/registration_abci/behaviours.py [hunk 1]
```diff
@@ -255,7 +255,7 @@ def get_addresses(self) -> Generator:  # pylint: disable=too-many-return-stateme
             return False
 
         # put service info in the shared state for p2p message handler
-        info: Dict[str, Dict[str, str]] = dict.fromkeys(registered_addresses)
+        info: Dict[str, Dict[str, str]] = {i: {} for i in registered_addresses}
```
Comment (DavidMinarsch) [inline packages/valory/skills/registration_abci/behaviours.py:258]: why the change?

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]

File: packages/valory/skills/registration_abci/behaviours.py (hunk 1 above)
Comment (solarw) [reply packages/valory/skills/registration_abci/behaviours.py:258]: @DavidMinarsch to create default value as `{}`, not `None`

Comment (Adamantios) [reply packages/valory/skills/registration_abci/behaviours.py:258]:
```suggestion
        info: Dict[str, Dict[str, str]] = dict.fromkeys(registered_addresses, {})
```

**Review by DavidMinarsch — APPROVED** [review]
